examples/general-plugins/CavityWriter/cavity.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<mardyn version="20100525" >
+	
```

Comment:
Did you take parameters from a paper? If not, please indicate clearly, that the simulation parameters are (arbitrarily?) chosen and the simulation will probably not lead to a cavitation process.

Reply:
Indeed no actual cavitation. 
Clarified here 92b4315